src/Aspire.Hosting.AWS/README.md

```diff
@@ -157,6 +157,102 @@ builder.AddProject<Projects.Frontend>("Frontend")
        .WithReference(construct, c => c.Url, "Url");
 ```
 
+## Integrating AWS Lambda Local Development
+
+You can develop and test AWS Lambda functions locally within your .NET Aspire application. This enables testing Lambda functions alongside other application resources during development.
+
+![alt text](image.png)
+
+### Adding Lambda Functions
+
+To add a Lambda function to your .NET Aspire AppHost, use the `AddAWSLambdaFunction` method. The method supports both executable Lambda functions and class library Lambda functions:
+
+```csharp
+// Add an executable Lambda function
+builder.AddAWSLambdaFunction<Projects.ExecutableLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ExecutableLambdaFunction")
+    .WithReference(awsConfig);
+
+// Add a class library Lambda function
+builder.AddAWSLambdaFunction<Projects.ClassLibraryLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ClassLibraryLambdaFunction::ClassLibraryLambdaFunction.Function::FunctionHandler")
+    .WithReference(awsConfig);
+
+```
+
+The handler parameter specifies the Lambda handler in different formats depending on the project type:
+
+- For executable projects: specify the assembly name.
+- For class library projects: use the format `{assembly}::{type}::{method}`.
+
+### API Gateway Local Emulation
+
+To add an API Gateaway emulator to your .NET Aspire AppHost, use the `AddAPIGatewayEmulator` method. 
+
+```csharp
+// Add Lambda functions
+var rootWebFunction = builder.AddAWSLambdaFunction<Projects.WebApiLambdaFunction>(
+    "RootLambdaFunction", 
+    handler: "WebApiLambdaFunction");
+
+var addFunction = builder.AddAWSLambdaFunction<Projects.WebAddLambdaFunction>(
+    "AddLambdaFunction", 
+    handler: "WebAddLambdaFunction");
+
+// Configure API Gateway emulator
+builder.AddAPIGatewayEmulator("APIGatewayEmulator", APIGatewayType.HttpApi)
+    .WithReference(rootWebFunction, Method.Get, "/")
+    .WithReference(addFunction, Method.Get, "/add/{x}/{y}");
+```
+
+The `AddAPIGatewayEmulator` method requires:
+
+- A name for the emulator resource
+- The API Gateway type (`HttpApi` or `RestApi`)
+
+Use the `WithReference` method to connect Lambda functions to HTTP routes, specifying:
+
+- The Lambda function resource
+- The HTTP method
+- The route pattern
+
+## Integrating Amazon DynamoDB Local  
```

Comment:
for dynamo stuff i just copied from norms blog post https://aws.amazon.com/blogs/developer/integrating-aws-with-net-aspire/

Reply:
I would rephrase this without the images so the README file looks okay when displayed on NuGet or in Visual studio package manager. You might be able to get the images to work on this other environments if you package the images but probably not worth it right now.

Reply:
i removed the images. i kept the wording the same since it still read fine to me